lib/dpd_api/request_handler.rb

```diff
@@ -28,8 +28,8 @@ def client
         wsdl: config.service_wsdl[request_name],
         endpoint: config.soap_endpoint[request_name],
         namespace: config.soap_endpoint[request_name],
-        open_timeout: 600,
-        read_timeout: 600,
+        open_timeout: config.open_timeout || 10, # Seconds
+        read_timeout: config.read_timeout || 30, # Seconds
```

Comment:
Generally, DPD API responds pretty quickly - usually ~1s or less: [Datadog](https://app.datadoghq.com/apm/traces?query=%40_top_level%3A1%20env%3Aproduction%20%40network.destination.ip%3Apublic-ws.dpd.com&agg_m=%40duration&agg_m_source=base&agg_t=max&analyticsOptions=%5B%22line%22%2C%22dog_classic%22%2Cnull%2Cnull%5D&cols=service%2Cresource_name%2C%40http.url%2C%40duration%2C%40http.method%2C%40http.status_code%2C%40_span.count%2C%40_duration.by_service&fromUser=true&graphType=flamegraph&historicalData=true&messageDisplay=inline&query_translation_version=v0&shouldShowLegend=true&sort=time&spanType=service-entry&traceQuery=&view=spans&viz=timeseries&start=1719398668677&end=1720608268677&paused=false)

it's only in the recent spike that this went up to several minutes